generators/app/templates/src/app/screens/OnBoarding/components/Swiper/components/Footer/buttonsInfo.js

```diff
@@ -0,0 +1,35 @@
+import i18next from 'i18next';
+
+export const screensButtonInfo = ({ nextScreen, skip, previousScreen, screenName }) =>
```

Comment:
use function naming conventions for `nextScreen`, `skip` and `previousScreen`. maybe `onSkip` `onNextScreen` `onPrevScreen` 🤷‍♂️